website/docs/r/synapse_workspace.html.markdown

```diff
@@ -183,6 +183,8 @@ The following arguments are supported:
 
 * `sql_administrator_login_password` - (Optional) The Password associated with the `sql_administrator_login` for the SQL administrator. If this is not provided `aad_admin` or `customer_managed_key` must be provided.
 
+* `azure_ad_only_authentication_enabled` - (Optional) Is Azure Active Directory Authentication the only way to authenticate with resources inside this synapse Workspace. Defaults to `false`.
```

Comment:
we have `azuread_authentication_only` in the provider elsewhere so we might want to use that?